modules/table/src/integrationTest/java/org/apache/ignite/distributed/ItTablePersistenceTest.java

```diff
@@ -197,7 +197,7 @@ public void afterSnapshot(RaftGroupService service) throws Exception {
     @Override
     public BooleanSupplier snapshotCheckClosure(JraftServerImpl restarted, boolean interactedAfterSnapshot) {
         MvPartitionStorage storage = getListener(restarted, raftGroupId()).getStorage();
-        Map<ByteBuffer, RowId> primaryIndex = getListener(restarted, raftGroupId()).getPk();
+        Map<ByteBuffer, RowId> primaryIndex = null; // getListener(restarted, raftGroupId()).getPk();
```

Comment:
This will cause NPE in line 206 below.

Reply:
this test is disabled anyway. I could adjust this code, but I don't know how to debug it since it's broken